VTEX - Logistics API.json

```diff
@@ -53,7 +53,643 @@
+                                                    "dayOfWeek": {
+                                                        "type": "integer",
+                                                        "description": "Day of the week from `0` (Sunday) to `6` (Saturday)."
+                                                    },
+                                                    "timeLimit": {
+                                                        "type": "string",
+                                                        "description": "Time limit in `hh:mm:ss` format."
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "cubicWeightSettings": {
+                                            "type": "object",
+                                            "description": "Measure that accounts package's volume, and not only weight.",                                            
+                                            "properties": {
+                                                "volumetricFactor": {
+                                                    "type": "number",
+                                                    "description": "Factor that will be used on the cubic weight calculation."
+                                                },
+                                                "minimunAcceptableVolumetricWeight": {
+                                                    "type": "number",
+                                                    "description": "Packages' minimum acceptable [volumetric weight](https://help.vtex.com/en/tutorial/understanding-the-cubic-weight-factor--tutorials_128)."
+                                                }
+                                            }
+                                        },
+                                        "modalSettings": {
+                                            "type": "object",
+                                            "description": "Configurations for the [modal](https://help.vtex.com/en/tutorial/how-does-the-modal-work--tutorials_125), which is the attachement of a specific product to a carrier specialized in delivering that type of product.",                                                                    
+                                            "properties": {
+                                                "modals": {
+                                                    "type": "array",
+                                                    "description": "Type of special freight. The possible values are: \r\n- `CHEMICALS` \r\n- `ELECTRONICS` \r\n- `FURNITURE` \r\n- `GLASS` \r\n- `LIQUID` \r\n- `MATTRESSES` \r\n- `REFRIGERATED` \r\n- `TIRES` \r\n- `WHITE_GOODS` \r\n- `FIREARMS`.",                              
```

Comment:
If there is already an enum, it's not necessary to have the possible values in the description